concepts/numbers/introduction.md

```diff
@@ -0,0 +1,53 @@
+# Introduction
+The built-in number types in C++ can be divided into integers and floating points.
+Integers are whole numbers like `0`, `691`, or `-2`.
+Examples of floating point numbers are `6.02214076`, `0.1`, or `-1.616`.
+
+## Integers
```

Comment:
Students coming from some other languages might expect `int`s to hold arbitrarily large integers. Suggestion: add a note about bounds.

Reply:
I added it as a section of the about.md, as it is not necessary to solve the exercise.
I plan to to a special concept on unsigned, limits, bit-width etc. So there they can experience it.